tests/test_integer_callback_arg.py

```diff
@@ -0,0 +1,18 @@
+from airflow import DAG
```

Comment:
Tests should also have copyright header. We actually should have tests about that.

Reply:
Well...I actually would like to have some consultation about them, but so far let's stick with ASF rules. At least, these guys are on the wave about legal bits and those are pretty clear.